src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -1,13 +1,63 @@
+from typing import Any
+
+from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+from drf_spectacular.utils import OpenApiExample, extend_schema_serializer
 from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
 
+from openforms.api.fields import FormioVariableKeyField
+from openforms.api.utils import get_from_serializer_data_or_instance
 from openforms.template.validators import DjangoTemplateValidator
 from openforms.utils.mixins import JsonSchemaSerializerMixin
 from openforms.utils.validators import validate_rsin
 
 
+class VersionChoices(models.IntegerChoices):
+    V1 = 1
```

Comment:
```suggestion
    V1 = 1, _("v1, template based")
```
etc. 

give some love to the humans reading the documentation :)